sphinx-docs/installguide/install_main.rst

```diff
@@ -0,0 +1,10 @@
+KA Lite Installation Guide
+===========================
+What a wonderful page. hELLO WELCOME YOU WILL LEARN HAO 2 INSTALL HERE
+
+.. toctree::
+	:maxdepth: 3
+	
+	Windows Installation Guide <install_windows>
```

Comment:
These files don't yet exists, and so when the docs are made it just silently fails to insert a link. Could be good to stub them out (just a title or something).